### PR TITLE
Treasurer role definition

### PR DESCRIPTION
**Pull request by Flaxscrip**

### CODE REVIEW

File: Roles/Treasurer/README.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
 # Treasurer
 
-TBD
\ No newline at end of file
+1. The Treasurer [role](..) is responsible for representing the donors/supporters/investors for a given project or task.
+1. The Treasurer (or their delegate) is responsible for:
+* publishing a BTC address for each task that is part of a project [Bounty](../../Projects/Bounties)
+* releasing the bounty to the [KeyMaster](../Keymaster) only when the Gatekeeper marks the task as completed.
+* reviewing a bounty's estimated value against comparable work in other markets.
+* periodically adjusting bounty sizing estimate table.
```
Comment (macterra) [inline Roles/Treasurer/README.md]: Maybe the Keymaster should be responsible for bounty bids attached to estimates. Or are you saying the Treasurer should be responsible for converting bids from USD to BTC?

File: Roles/Treasurer/README.md [hunk 2]
```diff
@@ -1,3 +1,9 @@
 # Treasurer
 
-TBD
\ No newline at end of file
+1. The Treasurer [role](..) is responsible for representing the donors/supporters/investors for a given project or task.
```
Comment (macterra) [inline Roles/Treasurer/README.md]: Nice

File: Roles/Treasurer/README.md (hunk 1 above)
Comment (Flaxscrip) [reply Roles/Treasurer/README.md]: There are 2 steps in the estimation process: sizing in terms of effort (S, M, L, XL) and pricing in terms of currency (x BTC). The pricing step needs comparables in the market. The Treasurer seems to be the right role for a check & balance to the KeyMaster on the market valuation of the bounty.

Comment (macterra) [reply Roles/Treasurer/README.md]: OK for now

**Pull request merged**